tests/test_class_image.py

```diff
@@ -49,8 +51,7 @@ def testGetLayerDiffIds(self):
 
     def testGetLayerObject(self):
         self.image2.load_image()
-        self.assertEqual(
-             self.image2.get_layer_object('123abc'), self.image2.layers[0])
+        self.assertEqual(self.image2.get_layer_object('123abc'), self.image2.layers[0])
```

Comment:
This line is longer than 79 characters when it's all on one line. It was broken up before to adhere to PEP8 style standards. Probably should revert this change :)